scenarios/hci/roles.yaml

```diff
@@ -0,0 +1,258 @@
+---
+###############################################################################
+# File generated by TripleO
```

Comment:
probably remove this comment to prevent any confusion? we 'generated' this ourselves via copy paste :)

Reply:
done :)